todoapp/src/components/Todo.vue

```diff
@@ -1,40 +1,210 @@
 <!-- eslint-disable vue/multi-word-component-names -->
-
 <script setup lang="ts">
-import type { TextString } from '../types/text'
+import type { TodoTypes } from '../types/text'
```

Comment:
Commonly the type names are singular and starts whit a capital letter so IMO a better name for this could be `Todo`